### ISSUE TITLE
Multivariate Laplace

### ISSUE THREAD

**Issue by patrickltobing** [LaplacesDemonR/LaplacesDemon#17]
Hi,

on line 1366 of file distributions.R, of the function dmvl(),

`(log(pi) - log(2) + log(2*z)*0.5)*0.5 - log(2*z)*0.5 -`

should be changed into 

`(log(pi) - log(2) + log(2*z)*0.5)*0.5 - sqrt(2*z) -`

because the fourth term above is originally within exp(),
so it returns the inside of it by applying log

and

on line 1382 of file distributions.R, of the function rmvl(),

`x <- mu + sqrt(e)*z`

should be

`x <- mu + sqrt(2e)*z`

because: a + b*sqrt(2v)*z ~ Laplace(a,b), where v ~ Exp(1) and z ~ N(0,1)

**Comment by singmann** [comment]
Can someone else confirm that this is correct and/or provide a pull request? I am happy to accept one.

**Comment by singmann** [comment]
Or can you provide a reference so I can check the logic?

**Comment by patrickltobing** [comment]
Yes, it is actually written on the web documentation as referred to the respective paper. [Eltoft, 2006]
[web_doc](https://www.rdocumentation.org/packages/LaplacesDemon/versions/16.1.1/topics/dist.Multivariate.Laplace)

**Comment by singmann** [comment]
Thanks a lot for this. I will push the new version with this fix to `CRAN` today.
